rcl/include/rcl/validate_security_context_name.h

```diff
@@ -64,10 +64,10 @@ rcl_validate_security_context_name(
 
 /// Deterimine if a given security context name is valid.
 /**
- * This is an overload with an extra parameter for the length of security_context.
- * \param[in] security_context The number of characters in security_context.
+ * \sa This is an overload of \ref rcl_validate_security_context_name with an extra parameter
```

Comment:
@ivanpauno same, I think you want to drop `\sa` here.

Reply:
Solved in https://github.com/ros2/rcl/pull/515/commits/77aad14964b484c6091bebe0cec28169546303a9.